### PR TITLE
better error message

### PR DESCRIPTION
**Pull request by ttimbers**

### CODE REVIEW

File: sqrt.py [hunk 1]
```diff
@@ -13,10 +13,13 @@
 
 opt = docopt(__doc__)
 
+
 def main(number):
-  number = int(number)
-  print(math.sqrt(number))
-    
+    if number < 0:
+        raise Exception("n should not a positive number")
```
Comment (mds-octocat) [inline sqrt.py]:
```suggestion
        raise Exception("n should be a positive number")
```

**Review by mds-octocat — APPROVED** [review]
Nice work, just one small change!

**Review by ttimbers — COMMENTED** [review]
Nice change

**Review by mds-octocat — APPROVED** [review]

**Comment by mds-octocat** [comment]
🎉

**Pull request merged**